docs/app/docs/cli_reference/devbox_create.md

```diff
@@ -0,0 +1,39 @@
+# devbox create
+
+Initialize a directory as a devbox project using a template
+
+## Synopsis
+
+Initialize a directory as a devbox project. This will create an empty devbox.json in the current directory. You can then add packages using `devbox add`
+
+```bash
+devbox create [dir] --template <template> [flags]
+```
+
+## List of templates
+
+* [**go**](https://github.com/jetpack-io/devbox/tree/main/examples/development/go)
+* [**node-npm**](https://github.com/jetpack-io/devbox/tree/main/examples/development/nodejs/nodejs-npm/)
+* [**node-typescript**](https://github.com/jetpack-io/devbox/tree/main/examples/development/nodejs/nodejs-typescript/)
```

Comment:
This example is outdated. I can update this later

Reply:
Since we clone them, we can update them at any point.